### ISSUE TITLE
Issues using with jest and functional components in react-intl 3

### ISSUE THREAD

**Issue by solomon23** [akameco/babel-plugin-react-intl-auto#78]
I get two crashes just in jest/enzyme when using Functional Components and react-intl3.

```
    TypeError: Cannot read property 'key' of undefined

      at NodePath._getKey (node_modules/@babel/traverse/lib/path/family.js:186:25)
      at NodePath.get (node_modules/@babel/traverse/lib/path/family.js:178:17)
      at node_modules/babel-plugin-react-intl-auto/lib/index.js:360:16
      at NodePath.find (node_modules/@babel/traverse/lib/path/ancestry.js:49:9)
      at addIdToFormatMessage (node_modules/babel-plugin-react-intl-auto/lib/index.js:359:18)
      at PluginPass.CallExpression (node_modules/babel-plugin-react-intl-auto/lib/index.js:424:9)
      at newFn (node_modules/@babel/traverse/lib/visitors.js:195:21)
      at NodePath._call (node_modules/@babel/traverse/lib/path/context.js:53:20)
      at NodePath.call (node_modules/@babel/traverse/lib/path/context.js:40:17)
      at NodePath.visit (node_modules/@babel/traverse/lib/path/context.js:88:12)
```

If I add the following code i get past that (line 360)

```
if (arg && arg.node && arg.get('key') && arg.get('key').node) {
      return arg.get('key').node.name === 'id';
    }
```

and then i get this error

```
    TypeError: properties[Symbol.iterator] is not a function

      at addIdToFormatMessage (node_modules/babel-plugin-react-intl-auto/lib/index.js:376:54)
      at PluginPass.CallExpression (node_modules/babel-plugin-react-intl-auto/lib/index.js:428:9)
      at newFn (node_modules/@babel/traverse/lib/visitors.js:195:21)
      at NodePath._call (node_modules/@babel/traverse/lib/path/context.js:53:20)
      at NodePath.call (node_modules/@babel/traverse/lib/path/context.js:40:17)
      at NodePath.visit (node_modules/@babel/traverse/lib/path/context.js:88:12)
      at TraversalContext.visitQueue (node_modules/@babel/traverse/lib/context.js:120:16)
      at TraversalContext.visitSingle (node_modules/@babel/traverse/lib/context.js:92:19)
      at TraversalContext.visit (node_modules/@babel/traverse/lib/context.js:148:19)
      at Function.traverse.node (node_modules/@babel/traverse/lib/index.js:96:17)
```

which i can fix by adding this line of code (line 376)

```
if (!properties[Symbol.iterator]) {
      return
   }
```

Let me know if you'd like a pull

**Comment by solomon23** [comment]
I should not I did fork your branch and saw that the latest code in master actually fixes this.  I also see these errors in my runtime code - not just react.

**Comment by akameco** [comment]
@solomon23 Please show me the reproduction repository.

**Comment by solomon23** [comment]
@akameco I really tried to build a minimal repro for you but wasn't able to do it in a reasonable amount of time.  I did track the issue down to that it's happening when that component imports another another component that imports a component that uses `injectIntl`.  If i remove that things start to compile.

Do you plan to release what's in your master branch soon here ? That does fix me up

**Comment by akameco** [comment]
Fixed #110
